Jenkinsfile

```diff
@@ -1,6 +1,6 @@
-env.ANSIBLE_SSH_CONTROL_PATH = "%(directory)s/%%h-%%r"
-env.ANSIBLE_HOST_KEY_CHECKING = false
+env.PYTHONUNBUFFERED = 1
 env.ANSIBLE_FORCE_COLOR = true
```

Comment:
Can we move this into ansible.cfg file as well

Reply:
i tried but i didn't get the expected result while tested

Reply:
trying again ...

Reply:
![bitmoji](https://render.bitstrips.com/v2/cpanel/614b3d25-fada-4a1f-8a0a-43a7cf2242ac-b368c035-efce-4f5f-b15b-0bdeb86fd3f9-v1.png?transparent=1&palette=1&width=246)